debian/rules

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/make -f
+
+include /usr/share/postgresql-common/pgxs_debian_control.mk
+
+override_dh_auto_build:
+	+pg_buildext build build-%v
+
+override_dh_auto_clean:
+	# +pg_buildext clean build-%v
```

Comment:
In one of other product's debian/rules file, I remember that we had a comment that reasons why we commented out this line (`# +pg_buildext clean build-%v`).

Can we copy paste that reasoning to here as well ?